src/libNode/PoWProcessing.cpp

```diff
@@ -293,10 +293,7 @@ bool Node::SendPoWResultToDSComm(const uint64_t& block_num,
     const auto& blocktype = get<BlockLinkIndex::BLOCKTYPE>(bl);
     if (blocktype == BlockType::DS) {
       uint16_t lastBlockHash = DataConversion::charArrTo16Bits(
-          m_mediator.m_dsBlockChain.GetLastBlock()
-              .GetHeader()
-              .GetHashForRandom()
-              .asBytes());
+          m_mediator.m_dsBlockChain.GetLastBlock().GetBlockHash().asBytes());
```

Comment:
This is the issue, kindly revert this part.

Reply:
yes, will revert. But even that had not helped.